dev-tools/missing-doclet/src/main/java/org/apache/lucene/missingdoclet/MissingDoclet.java

```diff
@@ -430,6 +430,11 @@ private void error(Element element, String message) {
     fullMessage.append(element.getKind().toString().toLowerCase(Locale.ROOT));
     fullMessage.append("): ");
     fullMessage.append(message);
-    reporter.print(Diagnostic.Kind.ERROR, element, fullMessage.toString());
+    if (((element.getKind() == ElementKind.PACKAGE) && (fullMessage.toString().contains("javadocs are missing")))){
```

Comment:
Shouldn't reporter.print(kind, String) be used for everything package-related? Why limit to "javadocs are missing"? I suspect other elements will also emit an NPE.

Can you post the stack trace from that NPE to jira too? I'm sure jdk folks will be interested.

Reply:
I am all for making this change, I just didn't have much familiarity with this code base, so was trying to only change it to get my specific issue fixed.   

I have added to the JIRA the stack trace.